.gitmodules

```diff
@@ -0,0 +1,8 @@
+[submodule "submodules/Atlus-Script-Tools"]
```

Comment:
We should think about changing the subdir for external dependencies to "external"